packages/documentation/src/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -17,6 +17,7 @@ const meta: Meta = {
     innerHTML: 'Hi there 👋',
     backgroundColor: 'primary',
     placement: 'top',
+    arrow: false,
```

Comment:
```suggestion
    arrow: null,
```

If the default of the component is "true", not setting it via story attributes or setting it to "true", should result in a value of "true" (arrow is visible).
If we set the arrow property to "false" the arrow should not be visible.